src/Components/BehaviorsModal/Internal/Widgets/DataHistoryWidget/DataHistoryWidget.tsx

```diff
@@ -212,32 +212,46 @@ const DataHistoryWidget: React.FC<IDataHistoryWidgetProps> = ({
 
     return (
         <div className={classNames.root}>
-            <div className={classNames.header}>
-                <span className={classNames.title}>{displayName}</span>
-                {mode === BehaviorModalMode.viewer && (
-                    <>
-                        {renderQuickTimePickerItem()}
-                        <OverflowMenu {...menuProps} />
-                    </>
-                )}
-            </div>
+            {errors.length > 0 ? (
+                <>
+                    <div className={classNames.header}>
+                        <span className={classNames.title}>{displayName}</span>
+                    </div>
+                    <DataHistoryWidgetErrorHandling
+                        errors={errors}
+                        styles={styles}
```

Comment:
why do we pass the entire styles object? SHouldn't this be a subcomponent style so that it only contains teh styles for the child?

Reply:
ahh yeah, removed this and reading styles from parents subcomponent